documentation/reference.md

```diff
@@ -219,7 +219,13 @@ There are three different rules for validating dates:
 ##### Full dates (Award Dates and Transaction Dates)
 The ```Award Date``` **must** provide a full date, including year, month and day in YYYY-MM-DD format (e.g. 2017-04-02 for the 2nd April 2017).
 
-In some rare cases, an award date might also need to include the time of the grant, using a date-time format (e.g. 2017-04-02T16:45:00Z - a grant made at 4.45pm).
+In some cases, in order to allow easier export of 360Giving data from grant systems, an award date might also need to include the time of the grant, using a date-time format (e.g. 2017-04-02T16:45:00Z - a grant made at 4.45pm).
+
+
+**Note** - the time component of a grant is not significant to Award Dates or Transaction Dates, and there is no importance in which hour a grant was made. 
```

Comment:
I think this sentence leaves some points ambiguous. How about "the time component of Award Dates or Transaction Dates is not significant and **must** be ignored by applications." ?

Reply:
This isn't **must** guidance in my view. My suggestion for this text is
In some cases, award date data exported from grant systems includes the time of the grant, using a date-time format (e.g. 2017-04-02T16:45:00Z - a grant made at 4.45pm).
*Note* - the time component of a grant is not significant to Award Dates or Transaction Dates and may be omitted from applications.

Reply:
I think if we allow applications to use the time component then we could see some misleading behaviour, such as:
- Time components being shown in interfaces even though they're meaningless
- Ordering behaviour still using the time component, so if a list is organised by `Award Date` and then by Funder, we'd see the grants made by a funder that used the time component listed last (because they happened after 00:00:00) instead of in their correct place alphabetically

Making this a must hopefully would prompt anyone using the standard in an application to actively do something to discard the time component.

Reply:
Sorry Rob, your suggestion does not reflect what the committee discussed in the last meeting. I will also remind us all that it was Open Data Services recommendation to add the time component to version 1 of the standard to allow for easier publication. Your suggestion above contradicts that.

Reply:
"The time component is never significant in Award Dates or Transaction Dates. Applications *should* ignore the time component when processing grants data" @morchickit @stevieflow

Reply:
This is now updated in the branch to reflect the changes suggested here.